catalogs/catalogio.py

```diff
@@ -5,15 +5,15 @@
 logger = logging.getLogger("catalogs.catalogio")
 
 
-class CatalogTools(object):
+class CatalogTools:
     """Represent a catalog"""
 
     def __init__(self, source, text=False):
         try:
             self.oscal = self._load_catalog_json(source, text)
-        except (IOError, FileNotFoundError, json.decoder.JSONDecodeError) as e:
-            logger.error(f"Unable to load catalog {source}: {e}")
-            raise CatalogLoadError(f"Could not load catalog {source}") from e
+        except (IOError, FileNotFoundError, json.decoder.JSONDecodeError) as exc:
+            logger.error("Unable to load catalog %s: %s", source, exc)
```

Comment:
In logging, you usually want to use lazy formatting so that the strings aren't evaluated unless they need to be. Using an f-string literal here would mean that the string is always evaluated.